### DISCUSSION TITLE
HDD power management

### DISCUSSION THREAD

**Discussion by feliiik** [immich-app/immich#13104] [Q&A]
Hello guys, I'm relly impressed by Immich, but I'm struggling with the settings for the power management - I do not want to invest a lot for the beggining until I learn more, so I'm using rpi4, OMV, Immich installed via Portainer and externa USB HDD, which I would like to put into sleep while not using.

**Is it possible to set HDD for hibernation during inactivity? OMV has this function, do I need to change perhaps in Portainer?**

**Comment by bo0tzz** [comment]
Immich has nothing to do with this, it's the sort of thing you'd configure on the OS.
